libs/wire-api/src/Wire/API/Routes/Version.hs

```diff
@@ -100,6 +100,8 @@ versionInt V4 = 4
 versionInt V5 = 5
 versionInt V6 = 6
 versionInt V7 = 7
+versionInt V8 = 8
+versionInt V9 = 9
```

Comment:
this will conflict with https://github.com/wireapp/wire-server/pull/4356

but maybe that's fine?

Reply:
Yes, that's totally fine. I am aware of this. There's no way to avoid conflicts if we work in parallel on related stuff. I see the PR has been merged in the PR so I'll rebase this PR to resolve the conflicts.